rules/game_process.tex

```diff
@@ -124,8 +124,41 @@ \subsubsection{Invalid Goal}
 In these cases a goal is not scored (that is, the goal is ruled invalid), and the game will proceed with a Goal Kick (\cf \cref{sec:free_kick}).
 The head referee should also advise why the goal is invalid, such as by calling ``Not indirect'', ``Played with hands'' or ``No own goal''.
 
-\subsubsection{Indirect Kick}
-\label{sec:indirect_kick}
+\subsubsection{Indirect Kick - Champion Cup}
+\label{sec:indirect_kick_champion}
+
+A team may only score a goal by making a second contact with the ball within is own team from any affected start or restart into \texttt{playing}. 
+
+\begin{itemize}
+  \item A robot may not score a goal from a direct kick, including via deflections.
+  \item Kick considered to be indirect are \texttt{Kick-off}, \texttt{Kick-in}, \texttt{Pushing free kick} and \texttt{Penalty kick}.
+  \item The ball must be deliberately played-at a second time by another robot of its own team before a goal may be scored.
+    A deliberate play at the ball includes successfully kicking the ball, dribbling the ball (and subsequently leaving possession of the ball), or the goalkeeper playing at the ball with its hands.
+  \item If an indirect free kick is kicked directly into the opponent's goal, the goal does not count and a goal kick is awarded.
+  \item Note that an own-goal may always be scored without requiring an indirect kick.
+  \item Direct kick is allowed for the duration of the penalty kick, but this rule still applied after this kick. Contact in the penalty kick count among this rule.  
```

Comment:
I still need to think about that as it's kinda weird to talk indirect for penalty as it's not listed in the list.

Reply:
From the Penalty kick rules:
> The attacking robot (taking the penalty kick) may only score a goal if it touches the ball once.
> Once the attacking robot has touched the ball, it may not score a goal until another robot (from either team) touches the ball.

Maybe we can add the penalty kick in the list as: "or after an unsuccessful Penalty kick" or something in this way.